src/pages/ct-default/index.js

```diff
@@ -3,7 +3,15 @@ import { ResourceIds } from 'src/resources/ResourceIds'
 import CtDefaults from './form/CtDefaults'
 
 const CtIndex = () => {
-  return <ImmediateWindow datasetId={ResourceIds.CtDefaults} labelKey={'rmd'} Component={CtDefaults} height={700} />
+  return (
+    <ImmediateWindow
+      datasetId={ResourceIds.CtDefaults}
+      labelKey={'rmd'}
+      Component={CtDefaults}
+      height={680}
```

Comment:
please decrease the height a little bit and increase the width so the lookup labels are visible